supporting-material-uploads/minutes/20200923-min.html

```diff
@@ -1,12 +1,13 @@
 <!DOCTYPE html>
 <html lang='en'>
 <head>
+  <meta charset="utf-8">
   <title>W3C/OGC Maps for the Web workshop -- 23 Sep 2020</title>
-  <link type="text/css" rel="STYLESHEET" href="https://www.w3.org/StyleSheets/base.css">
-  <link type="text/css" rel="STYLESHEET" href="https://www.w3.org/StyleSheets/public.css">
-  <link type="text/css" rel="STYLESHEET" href="https://www.w3.org/2004/02/minutes-style.css">
+  <link rel="stylesheet" href="https://www.w3.org/StyleSheets/base.css">
+  <link rel="stylesheet" href="https://www.w3.org/StyleSheets/public.css">
+  <link rel="stylesheet" href="https://www.w3.org/2004/02/minutes-style.css">
+  <link rel="stylesheet" href="style.css">
   <meta content="W3C/OGC Maps for the Web workshop" name="Title">  
```

Comment:
What is `meta name=title`? Is this supposed to be a `meta name="description`?

Reply:
Not sure, but it's created by the generator (it's on other minutes created by the same tools, [for example](https://www.w3.org/2020/07/29-svg-minutes.html)). No harm done leaving it, I guess.